docs/Installing.asciidoc

```diff
@@ -99,13 +135,66 @@ You can do that in +/etc/openqa/openqa.ini+:
 httpsonly = 0
 --------------------------------------------------------------------------------
 
+=== Database
+
+Since verion _4.5.1512500474.437cc1c7_ of openQA, PostgreSQL is used as the
+default database.
+
+It is possible to configure other databases.
+To configure access to the chosen database in openQA,
+edit +/etc/openqa/database.ini+ and change the settings in the +[production]+
+section.
+
+The +dsn+ value format technically depends on the database type and is
+documented for PostgreSQL at
+https://metacpan.org/pod/DBD::Pg#DBI-Class-Methods[DBD::Pg]
+
+====== Example for connecting to local PostgreSQL database
+
+[source,ini]
+--------------------------------------------------------------------------------
+[production]
+dsn = dbi:Pg:dbname=openqa
+--------------------------------------------------------------------------------
+
+====== Example for connecting to remote PostgreSQL database
+
+[source,ini]
+--------------------------------------------------------------------------------
+[production]
+dsn = dbi:Pg:dbname=openqa;host=db.example.org
+user = openqa
+password = somepassword
+--------------------------------------------------------------------------------
+
+====== Older versions of openQA
+
+Older versions of openQA, like the one distributed on Leap 42.3, had support for
```

Comment:
I don't want documentation about the past. The documentation of old versions is in old versions of openQA-doc

Reply:
Leap 42.3 is the present of the "really stable" openSUSE, and this package distributed is this oldie.
Would you prefer a section for Leap 42.3 pitfalls?

Reply:
not here. This sounds more like the topic of a personal blog.